owasp-top10-2021-apps/a3/copy-n-paste/app/util/db.go

```diff
@@ -126,16 +126,16 @@ func InitDatabase() error {
 
 	dbConn, err := OpenDBConnection()
 	if err != nil {
-		errOpenDBConnection := fmt.Sprintf("OpenDBConnection error: %s", err)
+		errOpenDBConnection := ("OpenDBConnection error: %s" + err)
```

Comment:
I don't see any need to make any changes at this point. Is this change related to the vulnerability found?

Reply:
It has nothing to do with the vulnerability, I just understood that it would be a good practice in the code